src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliNative.cs

```diff
@@ -16,73 +16,74 @@
 
 namespace System.IO.Compression
 {
+    ///  /// <summary>
+    /// 0 - Process input. Encoder may postpone producing output, until it has processed enough input.
+    /// 1 - Produce output for all processed input.  Actual flush is performed when input stream is depleted and there is enoughspace in output stream.
+    /// 2 - Finalize the stream. Adding more input data to finalized stream is impossible.
+    /// 3 - Emit metadata block to stream.Stream is soft-flushed before metadata block is emitted. Metadata bloc MUST be no longer than than 16MiB.
```

Comment:
Typos: 
- enoughspace -> enough space
- than than -> than
- Emit metadata block to stream.Stream... -> Emit metadata block to stream. Stream...